packages/ui/src/components/Table.svelte

```diff
@@ -73,9 +73,15 @@
       {/if}
       <tbody>
         {#each data as doc (doc._id)}
-          <tr class="tr-body" on:click={() => dispatch('rowClick', { id: doc._id })}>
+          <tr class="tr-body" on:click={() => dispatch('rowClick', doc)}>
             {#each docToRow(doc) as cell}
-              <td><svelte:component this={cell.component} {...cell.props} /></td>
+              <td
+                ><svelte:component
+                  this={cell.component}
+                  {...cell.props}
+                  on:click={() => dispatch('rowClick', doc)}
```

Comment:
Why we need both events on a row and a celll?

Reply:
Good catch, we didn't.